Debian/8/templates/template_partitions_rights

```diff
@@ -0,0 +1,26 @@
+<def-group>
+  <definition class="compliance" id="mount_optionPOINTID_FLAGID" version="1">
```

Comment:
Ahhh.... Thanks @pthierry38. 

> If you prefer I can update the POINTID value to delete the starting _.

I am wondering if you should that way someone doesn't accidentally change the template.